### DISCUSSION TITLE
Sending a password reminder email

### DISCUSSION THREAD

**Discussion by barbudour** [strapi/strapi#7192] [Legacy Topics]
Hello. Is it possible to make a user password reminder functionality? I need the following functionality: if the user has forgotten his password, then he indicates his mail and receives the current password by mail

**Comment by lauriejim** [comment]
Hello! Sending the current password is not possible because we save a hash of the password. It's secured.
After that the forgotten / reset endpoint already exist.
Here is the documentation for that - https://strapi.io/documentation/v3.x/plugins/users-permissions.html#forgotten-password
